Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -62,39 +62,40 @@ namespace EMotionFX
                 {
                     editContext->Class<ActorComponent::BoundingBoxConfiguration>("Actor Bounding Box Config", "")
                         ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-
                         ->DataElement(AZ::Edit::UIHandlers::ComboBox, &ActorComponent::BoundingBoxConfiguration::m_boundsType,
                                       "Bounds type",
-                                      "The method used to compute the Actor bounding box. NOTE: ordered by least expensive to compute to most expensive to compute."
-                        )
-                        ->EnumAttribute(ActorInstance::BOUNDS_STATIC_BASED, "Static bounds (source-asset bounds)")
-                        ->EnumAttribute(ActorInstance::BOUNDS_NODE_BASED, "Bone position-based")
-                        ->EnumAttribute(ActorInstance::BOUNDS_NODEOBB_BASED, "Bone local bounding box-based")
-                        ->EnumAttribute(ActorInstance::BOUNDS_MESH_BASED, "Render mesh vertex position-based (VERY EXPENSIVE)")
-
-                        ->DataElement(0, &ActorComponent::BoundingBoxConfiguration::m_autoUpdateBounds,
+                                      "The method used to compute the Actor bounding box. NOTE: ordered by least expensive to compute to most expensive to compute.")
+                            ->EnumAttribute(ActorInstance::BOUNDS_STATIC_BASED, "Static (Recommended)")
+                            ->EnumAttribute(ActorInstance::BOUNDS_NODE_BASED, "Bone position-based")
+                            ->EnumAttribute(ActorInstance::BOUNDS_MESH_BASED, "Mesh vertex-based (Expensive)")
```

Comment:
As we know, mesh-based bbox calculation is expensive and the typical user should be *very* wary of using this option. 

In some extremely rare edge cases, this may be the only way to do get the desired result. although this PR should in theory eliminate the last of those edge cases.

Dissuading users from picking a "bad" option is difficult, but I think that all caps "VERY EXPENSIVE" communicates the danger better than "Expensive".

Reply:
Changed it back to caps